frontend/src/pages/Assessments/data/reducer.tsx

```diff
@@ -1,25 +1,35 @@
 import { RootStateOrAny } from "react-redux";
-import { Action, ActionTypes, AssessmentState } from "../../../models/types";
+import {
+  Action,
+  ActionTypes,
+  Assessment,
+  AssessmentState,
+} from "../../../models/types";
+
+const emptyAssessment: Assessment = { id: -1, currentIndex: 0, words: [] };
 
 const initialState: AssessmentState = {
-  words: [],
+  assessment: emptyAssessment,
```

Comment:
I will think to initialize id, currentIndex, words in this initialState instead of just put assessment here. So like
```
   const initalState: AssessmentState = {
    id: 1,
    currentIdx: 0,
   words: [],
}
```
Also wondering why is ID -1 here?